api/utils/hysds_util.py

```diff
@@ -633,17 +633,22 @@ def get_mozart_jobs(username, page_size=10, offset=0):
         :return:
         """
     params = dict()
-    params["page_size"] = page_size
-    params["offset"] = offset  # this is specifies the offset
+    logging.info("hysds utils")
```

Comment:
```suggestion
    logging.debug("hysds utils")
```
This does not need to log at level INFO

Reply:
Also why do we need this log line ? Can we remove it ?

Reply:
Removed